<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>267</width>
    <height>252</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>增加操作员</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/back.jpg);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="0">
    <widget class="QLineEdit" name="lineEdit_2"/>
   </item>
   <item row="6" column="0">
    <widget class="QComboBox" name="comboBox"/>
   </item>
   <item row="2" column="0">
    <widget class="QLineEdit" name="lineEdit"/>
   </item>
   <item row="3" column="0" alignment="Qt::AlignLeft">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>密码：</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" alignment="Qt::AlignLeft">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>权限设定：</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" alignment="Qt::AlignLeft">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>增加操作员</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" alignment="Qt::AlignHCenter">
    <widget class="QPushButton" name="comfirmButton">
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" alignment="Qt::AlignLeft">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>账号：</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0" alignment="Qt::AlignHCenter">
    <widget class="QPushButton" name="returnButton">
     <property name="text">
      <string>返回</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="background_image.qrc"/>
 </resources>
 <connections/>
</ui>
